tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-distributed-test-ci-pipeline.yml

```diff
@@ -25,7 +25,16 @@ jobs:
         -m
       DisplayName: 'Build'
 
-    # all distributed tests
+    # Entry point for all distributed CI tests.
```

Comment:
good info. i wonder if it's a little hard to find though. consider putting this somewhere more obvious, e.g., a readme in orttraining/orttraining/test/python.

Reply:
I added a md file with more detailed description.